packages/webapp/src/webview/ui/components/UIComponentsLib/Icons/Icons.tsx

```diff
@@ -21,6 +21,7 @@ export enum UiIcons {
     ArrowDown = 'ArrowDown',
     Association = 'Association',
     Bulb = 'Bulb',
+    Chat = 'Chat',
```

Comment:
Seem `Icons.ts` is the last file in folder `UIComponentsLib`. What is the difference to `@sap-ux/ui-components` -> Icons?
Could we consider merging our changes here back to `@sap-ux/ui-components`?

Reply:
In ui components we don't have this icon but I think we should move the icons we added over time to the open source and get rid of this file. I'll create a tbi and I'll talk to Goran. Thanks!